src/main/java/Game.java

```diff
@@ -18,6 +15,10 @@
 	private static Map map;
 	private static int PORT;
 	private boolean isBusy; //Spin lock in case 2 commands at same time.
+	private static String north = "north";
```

Comment:
I made these, but I am not sure how useful they are. There are places, such as in switch statements where I could not use these constants.